handler/src/test/java/io/netty/handler/ssl/SslUtilsTest.java

```diff
@@ -26,7 +28,7 @@
 import java.nio.ByteOrder;
 import java.security.NoSuchAlgorithmException;
 
-import static io.netty.handler.ssl.SslUtils.getEncryptedPacketLength;
+import static io.netty.handler.ssl.SslUtils.*;
```

Comment:
Please dont use wildcard imports

Reply:
Oh this project doesn't like wildcards? I searched for "\.\*" and saw enough that I thought this projected used them. I can remove it.